js/adapt-contrib-slider.js

```diff
@@ -371,10 +397,13 @@ define(function(require) {
 
         // according to given item index this should make the item as selected
         selectItem: function(itemIndex) {
+            this.$el.a11y_selected(false);
             _.each(this.model.get('_items'), function(item, index) {
                 item.selected = (index == itemIndex);
                 if(item.selected) {
                     this.model.set('_selectedItem', item);
+                    this.$('.confidenceSlider-scale-number[data-id="'+(itemIndex+1)+'"]').a11y_selected(true);
```

Comment:
I posed a question on this. As the confidence slider heavily extends slider and doesn't reimplement selectItem, this line is currently needed to make confidence slider accessible. I suggest that two separate PR's are made after this one. One to remove this line and one to update confidence slider with it's own implementation of selectItem.

Let me know your thoughts.